astropy/modeling/tests/test_physical_models.py

```diff
@@ -115,7 +115,8 @@ def test_blackbody_overflow():
     ):
         with np.errstate(all="ignore"):
             bb_lam = bb(wave) * u.sr
-    flux = bb_lam.to(photlam, u.spectral_density(wave * u.AA)) / u.sr
+    with np.errstate(all="ignore"):
+        flux = bb_lam.to(photlam, u.spectral_density(wave * u.AA)) / u.sr
```

Comment:
Eagerly computing the conversion factors means that this code will produce a warning (about division with zero), but if a different unit is used instead of `photlam` then it could produce a warning on current `main` too. The test here deliberately uses an invalid (zero) wavelength value and already suppresses multiple warnings, so one more warning suppression isn't a big deal.